packages/grpc-native-core/ext/call.h

```diff
@@ -98,7 +98,7 @@ class Op {
 };
 
 typedef std::vector<unique_ptr<Op>> OpVec;
-struct tag {
+struct tag : public Nan::AsyncResource {
```

Comment:
Why have `tag` inherit from `Nan::AsyncResource`, instead of just having a `Nan::AsyncResource` member?

Reply:
Other reason: `tag` represents an async operation, so it makes sense for it to be a subclass of `AsyncResource`.

Reply:
We don't explicitly delete any of the members. They get deleted by default when the tag object is deleted. And subclassing merely for the purpose of categorization isn't really compelling to me, so I would prefer to use composition here.

Reply:
Composition it is.